### PR TITLE
new type scale in MWC

### PR DESCRIPTION
**Pull request by mperrotti**
#### Related issues
Fixes https://meetup.atlassian.net/browse/SDS-282

#### Description
Using new type scale for our components

#### Screenshots (if applicable)
typescale

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -80,7 +80,7 @@
     "storybook-addon-i18n-tools": "1.0.0",
     "style-loader": "0.16.1",
     "swarm-icons": "0.2.2",
-    "swarm-sasstools": "1.4.15",
+    "swarm-sasstools": "2.0.14-beta",
```
Comment (akdetrick) [inline package.json]: we can use the real release now, `1.7.18`

Comment (mperrotti) [inline package.json]: Done 👍

**Comment by mperrotti** [comment]
This should get merged first - https://github.com/meetup/swarm-sasstools/pull/23

Then I'll upgrade `swarm-sasstools` and this one can get merged

**Comment by akdetrick** [comment]
Pull 23 of `swarm-sasstools` has been merged. Once the update is specified in this branch, I think we're good to merge.

**Comment by mperrotti** [comment]
Upgraded sasstools version with new type

**Comment by mperrotti** [comment]
Update after talking to Chris:
We want to have the new type scale (14,16,18,22...), but he doesn't want the type to scale with the viewport until he does some more exploration.

Going to make a PR to sasstools to prevent the type from growing with the viewport, upgrade here, then we can merge and at least use the new type scale.

Sound good?

**Comment by mperrotti** [comment]
Here's the sasstools PR: https://github.com/meetup/swarm-sasstools/pull/24

**Pull request merged**